src/styles/pages/PuzzlePage.module.scss

```diff
@@ -94,3 +91,45 @@
 .feedbackContainer {
   margin: 3rem 0;
 }
+
+/* MEDIA QUERIES*/
+@media (max-width: 600px) {
+  .card {
+    max-width: 600px;
+    flex-direction: column;
+  }
+}
+@media (max-width: 768px) {
```

Comment:
The breakpoints should be constants in another file
Also the media queries themselves should be mixins